common/Cargo.toml

```diff
@@ -46,6 +46,7 @@ thiserror = "1"
 uniffi = { version = "^0.19", optional = true }
 uniffi_macros = { version = "^0.19", optional = true }
 url = "2"
+once_cell="1"
```

Comment:
maybe this dependency can be moved under `[target.'cfg(not(target_arch = "wasm32"))'.dependencies]` ?

Reply:
ok